.golangci.yml

```diff
@@ -0,0 +1,28 @@
+linters:
+  enable-all: true
+  disable:
+    # all struct members initialized
+    - exhaustruct
+    # forbids fmt.Print.* and others
+    - forbidigo
+    # no global variables
+    - gochecknoglobals
+    # specify parallel on every test
+    - paralleltest
+    # tests in a separate package
+    - testpackage
+    # short variable names
+    - varnamelen
+    # whitespace linter
+    - wsl
+linters-settings:
+  lll:
```

Comment:
What does `lll` stand for?

Reply:
[line length linter](https://golangci-lint.run/usage/linters/#lll) I think

Reply:
if you want to use the default 120 that works too

Reply:
Thanks for the doc link, I couldn't find it and thought maybe it was a typo lol

100 or 120 are both fine by me.